### ISSUE TITLE
ForkPullRequestDiscoveryTrait is not discovered by JobDSL

### ISSUE THREAD

**Issue by cronik** [jenkinsci/bitbucket-branch-source-plugin#290]
### Your checklist for this issue

- [x ] Jenkins version

- [ x] Plugin version

- [ ] Bitbucket cloud

- [ ] Bitbucket server and version

### Description

* Jenkins version = 2.204.2
* Plugin version = 2.7.0

When browsing the Job-DSL api-viewer the `bitbucketForkDiscovery` symbol is not shown. 
<img width="374" alt="Screen Shot 2020-03-09 at 10 05 51 AM" src="https://user-images.githubusercontent.com/1683550/76220280-98632800-61ed-11ea-9ea0-a92f892535fb.png">

It does appear to find it when executing Job-DSL script but it's unclear how to configure the trust argument. I tried various combinations all without any luck.
```groovy
organizations {
        bitbucket {
           ...
            traits {
                bitbucketForkDiscovery {
                    strategyId(1)
                    trust { bitbucketTrustEveryone{} }
                    // bitbucketTrustEveryone{}
                    // trust(class: 'com.cloudbees.jenkins.plugins.bitbucket.ForkPullRequestDiscoveryTrait\$TrustEveryone')
                }
            }
        }
    }
```

The current workaround is to manually configure the traits
```groovy
organizationFolder('myfolder') {
  configure { node ->
        def traits = node / navigators / 'com.cloudbees.jenkins.plugins.bitbucket.BitbucketSCMNavigator' / traits
        traits << 'com.cloudbees.jenkins.plugins.bitbucket.ForkPullRequestDiscoveryTrait' {
            strategyId(1)
            trust(class: 'com.cloudbees.jenkins.plugins.bitbucket.ForkPullRequestDiscoveryTrait$TrustEveryone') 
        }
    }
}
```

**Comment by Jetersen** [comment]
known issue already reported previously and on Jira.

It's an issue in structs.

**Comment by Jetersen** [comment]
duplicate of [JENKINS-61119](https://issues.jenkins-ci.org/browse/JENKINS-61119) blocked by [JENKINS-26535](https://issues.jenkins-ci.org/browse/JENKINS-26535)
